README.md

```diff
@@ -18,7 +18,7 @@ The [Media Types](media-types.md) document is a starting point to understanding
 
 The high level components of the spec include:
 
-* An [image manifest](manifest.md) and [filesystem serialization](serialization.md) (base layer)
+* An [image manifest](manifest.md) and [filesystem serialization](config.md) (base layer)
```

Comment:
I think *this* “filesystem serialization” [should be “image config”][1].

[1]: https://github.com/opencontainers/image-spec/pull/255/files#r77894629